sparse_strips/vello_cpu/src/util.rs

```diff
@@ -4,6 +4,6 @@
 pub(crate) mod scalar {
     #[inline(always)]
     pub(crate) const fn div_255(val: u16) -> u16 {
-        (val + 1 + (val >> 8)) >> 8
```

Comment:
Can we link to e.g. #904 here? Given that this is u16->u16, we could also have a test verifying that this is:
1) On-average correct (that is, it isn't meaningfully biased, if that's a property this has).
2) No more than (e.g.) 1 away from the "correct" value for all values

We should also probably have some docs on this method.